config/core/200-serviceaccount.yaml

```diff
@@ -46,3 +46,18 @@ roleRef:
   kind: ClusterRole
   name: knative-serving-admin
   apiGroup: rbac.authorization.k8s.io
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: knative-serving-controller-addressable-resolver
+  labels:
+    serving.knative.dev/release: devel
+subjects:
+  - kind: ServiceAccount
+    name: controller
+    namespace: knative-serving
+roleRef:
+  kind: ClusterRole
+  name: addressable-resolver
```

Comment:
Is this something we install with serving?  Should we be linking this from some shared place? 🤔 

/assign @n3wscott

Reply:
It is true, you need to also creat the addressable-resolver cluster role binding and hope it stays in sync with eventings version

Reply:
So.. is this good as it is, or do we do need to do something else in serving?

Reply:
nag, nag @mattmoor @n3wscott I don't really understand what we should do here 😅 . The addressable-resolver cluster role is what this PR introduces. Is there something else we want to do or is this good-to-go?

Reply:
I think we need to include an instance of this ClusterRole, which at present is only defined in eventing IIUC.

Reply:
https://github.com/knative/eventing/blob/b3184ba9dcd0d0891a46ec88a7ddfd91f3ae12e9/config/core/roles/addressable-resolvers-clusterrole.yaml#L19

Reply:
hm. Im not sure how well that will work. We'd then both be defining the same ClusterRole. That's not the end of the world I guess but the current version in eventing has the `eventing.knative.dev/release: devel` label, whereas presumably the same role that went through the serving sausage-making-procedure would end up with a `serving.knative.dev/release` label 🤔. I guess that'll probably still apply cleanly, but it's a bit funky at least. Do we have any CI that'd tell us about any clashes that'd prevent both serving and eventing being installed at the same time?

If y'all both think that's fine you have way more context than I do so I'll just add this to the PR, but it feels a bit odd.

Reply:
I wish there were a better story here (we something similar with the image cache resource).  Personally I think we should drop the prefix on our version labels since we coversion things anyways.

Reply:
Heh, ok, let me add and push and we'll figure out the other stuff later

Reply:
Updated, Im adding a todo to go and properly read up on all this duck stuff because it looks super cool but I totes don't really understand what I just did :D. I hear there was a good talk on it recently I could start with!